server/ntb/tests/io/fixtures/stt_newsml_sport_test.xml

```diff
@@ -0,0 +1,107 @@
+</itemMeta>
+<contentMeta>
+<urgency>3</urgency>
+<contentCreated>2018-01-23T13:31:01+02:00</contentCreated>
+<contentModified>2018-01-23T13:31:29+02:00</contentModified>
+<altId type="sttidtype:textid">115007439</altId>
+<located type="loctyp:City">
+<name>Hämeenlinna</name>
+</located>
+<infoSource qcode="sttsource:1">
+<name>STT</name>
+</infoSource>
+<altId type="sttidtype:runningnumber" environment="sttcounter:daily">stt 123</altId>
+<rating value="4" scalemin="1" scalemax="9" scaleunit="rscaleunit:number" ratingtype="sttrating:webprio"/>
+<subject type="cpnat:abstract" literal="Related topic id" qcode="stt-topics:441649"/>
+<subject type="cpnat:abstract" qcode="sttdepartment:16">
+<name>Urheilu</name>
+</subject>
+<subject type="cpnat:abstract" qcode="sttsubj:15000000">
+<name>Urheilu</name>
+</subject>
+<subject type="cpnat:abstract" qcode="sttsubj:15031000">
+<name>Jääkiekko</name>
+</subject>
+<slugline/>
+<dateline>Hämeenlinna</dateline>
+<by/>
+<description role="drol:summary"/>
+<headline>Hämeen Sanomat: Ilmarinen hakee jääkiekkoseura Espoo Unitedia konkurssiin</headline><creditline>STT&#8211;</creditline>
+
+<genre type="sttnat:texttype" qcode="sttgenre:1" literal="Pääjuttu">
+<name>Pääjuttu</name>
+</genre>
+<genre type="sttnat:versiontype" qcode="sttversion:5" literal="Loppuversio">
+<name>Loppuversio</name>
+<related qcode="sttrel:actuallength" value="705"/>
+</genre>
+</contentMeta>
+<assert qcode="sttlocmeta:rich:8249">
+<name>Espoo</name>
+<definition/>
+<geoAreaDetails>
+<position latitude="60.208202" longitude="24.765419"/>
+</geoAreaDetails>
+<POIDetails>
+<address>
+<line> </line>
+<postalCode/>
+</address>
+</POIDetails>
+<broader type="cpnat:geoArea" qcode="sttcity:13">
+<name>Espoo</name>
+</broader>
+<broader type="cpnat:geoArea" qcode="sttstate:31">
+<name>Uusimaa</name>
+</broader>
+<broader type="cpnat:geoArea" qcode="sttcountry:1">
+<name>Suomi</name>
+</broader>
+<broader type="cpnat:geoArea" qcode="wldreg:150">
+<name>Eurooppa</name>
+</broader>
+</assert>
+<assert qcode="sttlocmeta:rich:8975">
+<name>Suomi</name>
+<definition/>
+<geoAreaDetails>
+<position latitude="61.490356" longitude="21.793088"/>
+</geoAreaDetails>
+<POIDetails>
+<address>
+<line> </line>
+<postalCode/>
+</address>
+</POIDetails>
+<broader type="cpnat:geoArea" qcode="sttcountry:1">
+<name>Suomi</name>
+</broader>
+<broader type="cpnat:geoArea" qcode="wldreg:150">
+<name>Eurooppa</name>
+</broader>
+</assert>
+<contentSet>
+<inlineXML contenttype="xhtml/xml">
+<html>
+<body>
+<p>Eläkevakuutusyhtiö <Company>Ilmarinen</Company> hakee jääkiekkoseura Espoo Unitedia konkurssiin, kertoo <a href="https://www.hameensanomat.fi/uutiset/kiekko/343838-ilmarinen-hakee-espoo-unitedia-konkurssiin">Hämeen Sanomat</a> nettisivuillaan. Lehden tietojen mukaan Ilmarinen on jättänyt konkurssihakemuksen Espoon käräjäoikeuteen tällä viikolla.</p>
```

Comment:
I think we have to somehow handle this `<Company>` tag, or is it stripped automatically?

Reply:
@petrjasek I've re-used the [parse_inline_content method from newms_2_0](https://github.com/superdesk/superdesk-core/blob/master/superdesk/io/feed_parsers/newsml_2_0.py#L223) but this method is actually only keeping the text part of the elements, which is wrong (and it's probably not correct for Newsml 2.0 too).

I've changed that for STT, it is now keeping the whole content and stripping any non HTML tag (but keeping content of the tag).

Thanks for the feedback!